src/client/styles/Consents.ts

```diff
@@ -10,29 +9,25 @@ import {
 export const heading = css`
   color: ${neutral[0]};
   margin: 0 0 ${space[3]}px;
-  ${headline.xxsmall({ fontWeight: 'bold' })};
-  font-size: 17px;
+  ${headline.xxxsmall({ fontWeight: 'bold' })};
 `;
 
 export const headingMarginSpace6 = css`
   margin-top: ${space[6]}px;
 `;
 
-export const headingWithMq = css`
-  ${heading}
-
-  ${from.tablet} {
-    ${headline.xxsmall({ fontWeight: 'bold' })}
-  }
-`;
-
 export const text = css`
   margin: 0;
   color: ${neutral[7]};
   ${textSans.medium()}
   max-width: 640px;
 `;
 
+export const textBold = css`
+  ${text};
+  ${textSans.medium({ fontWeight: 'bold' })};
```

Comment:
since you're already inheriting the font style, through the `${text}`, I think it's preferable just to update the weight to `bold`
```suggestion
  font-weight: bold;
```